docs/index.md

```diff
@@ -1,17 +1,78 @@
-# Welcome to MkDocs
+# Melange
 
-For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+## OCaml for JavaScript developers
 
-## Commands
+---
 
-* `mkdocs new [dir-name]` - Create a new project.
-* `mkdocs serve` - Start the live-reloading docs server.
-* `mkdocs build` - Build the documentation site.
-* `mkdocs -h` - Print help message and exit.
+Melange is a fork of the OCaml compiler that compiles to JavaScript. Melange
```

Comment:
I'd say "Melange is a backend for the OCaml compiler that emits JavaScript.

Reply:
Updated in https://github.com/melange-re/documentation-site/commit/21276ef408cfbdecda631b5dced72f23b92a79da. I also replaced other appearances of the sentence in "Learn".